plexpy/config.py

```diff
@@ -25,7 +25,7 @@ def bool_int(value):
     'PMS_NAME': (unicode, 'PMS', ''),
     'PMS_PORT': (int, 'PMS', 32400),
     'PMS_TOKEN': (str, 'PMS', ''),
-    'PMS_SSL': (int, 'General', 0),
+    'PMS_SSL': (int, 'PMS', 0),
```

Comment:
If `pms_ssl` is set to `1` under `[General]`, then it doesn't carry over to `[PMS]`.